PSModule/Helper.cs

```diff
@@ -429,7 +429,7 @@ public static void CreateFailedStepsReport(IDictionary<string, IList<ReportMetaD
                 table.RenderControl(new HtmlTextWriter(sw));
                 html = sw.ToString();
             }
-            File.WriteAllText($@"{uftWorkingFolder}\res\Report_{buildNumber}\Failed tests", html);
+            File.WriteAllText($@"{rptPath}\{FAILED_TESTS}", html);

```

Comment:
It is recommended to use `Path.Combine` method instead of concatenating the path by string.